test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -190,22 +186,22 @@ TEST_F(AmetsuchiTest, SampleTest) {
   iroha::Amount amount;
 
   // Block 1
-  auto block1 =
-      TestBlockBuilder()
-          .transactions(std::vector<shared_model::proto::Transaction>(
-              {TestTransactionBuilder()
-                   .creatorAccountId("admin1")
-                   .createRole(
-                       "user",
-                       std::set<std::string>{
-                           can_add_peer, can_create_asset, can_get_my_account})
-                   .createDomain(domain, "user")
-                   .createAccount(user1name, domain, fake_pubkey)
-                   .build()}))
-          .height(1)
-          .prevHash(fake_hash)
-          .txNumber(1)
-          .build();
+  auto block1 = TestBlockBuilder()
+                    .transactions(std::vector<shared_model::proto::Transaction>(
+                        {TestTransactionBuilder()
```

Comment:
it is better to assign this to a variable, to improve readability.

Reply:
This change happens because of clang-format. We have another task for ametsuchi test improvement, so I think it is out of scope of the PR.